frontend/src/metabase/styled-components/theme/css-variables.ts

```diff
@@ -41,9 +41,10 @@ export function getMetabaseSdkCssVariables(theme: MantineTheme, font: string) {
   return css`
     :root {
       --mb-default-font-family: ${font};
+      ${createColorVars("light")}
       ${getSdkDesignSystemCssVariables(theme)}
-      ${getDynamicCssVariables(theme)}
       ${getThemeSpecificCssVariables(theme)}
+      ${getDynamicCssVariables(theme)}
```

Comment:
Sort the color order the same way we sort on the core app.
https://github.com/metabase/metabase/blob/0760630f106ff9796184527ab06ae50f84486b70/frontend/src/metabase/styled-components/theme/css-variables.ts#L32-L35